master/getting-started/kubernetes/tutorials/using-calicoctl.md

```diff
@@ -2,53 +2,13 @@
 title: Using calicoctl in Kubernetes
 ---
 
-There are two ways to run `calicoctl` in Kubernetes:
+`calicoctl` provides more commands and functionality than `kubectl`. 
```

Comment:
> +`calicoctl` provides more commands and functionality than `kubectl`. 

This statement isn't really true. calicoctl provides a different set of commands than kubectl.

Reply:
OK, makes sense.